### PR TITLE
Remove unused framer motion features

### PR DESCRIPTION
**Pull request by mezotv**
## 📝 Description

> Changes the motion import to the more lightweigth m component. 

## ⛳️ Current behavior (updates)

> Currently uses the motion import which loads all of the features even the ones you don't need.

## 🚀 New behavior

> Uses the lightweight m component with only the animation features loaded into. Since thats the only thing the components use.

## 💣 Is this a breaking change (Yes/No):

> Not a breaking change!

## 📝 Additional Information

Docs for this change: https://www.framer.com/motion/guide-reduce-bundle-size/

### CODE REVIEW

**Comment by mezotv** [comment]
New pull request as the old one was force closed by github. I will add the changeset later aswell as fixing the tabs bug! @jrgarciadev

**Comment by mezotv** [comment]
Hey @jrgarciadev would you mind checking my changeset. Im not sure if I did it right, got a bit confused by the way it works.

**Comment by mezotv** [comment]
The tabs component should work again. It required the other domMax feature.

File: .changeset/warm-zoos-fry.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+"@nextui-org/accordion": major
+"@nextui-org/use-aria-accordion-item": major
+"@nextui-org/modal": major
+"@nextui-org/navbar": major
+"@nextui-org/popover": major
+"@nextui-org/ripple": major
+"@nextui-org/tabs": major
+"@nextui-org/tooltip": major
```
Comment (wingkwong) [inline .changeset/warm-zoos-fry.md:9]: This PR doesn't include any breaking changes. I believe you can just use `patch` instead of `major`.

Comment (mezotv) [reply .changeset/warm-zoos-fry.md:9]: Yea I thought so too buy pnpm changeset made them all major.

Comment (wingkwong) [reply .changeset/warm-zoos-fry.md]: You can press enter to skip. You should see `No items were selected`. For patch, just simply press enter twice. e.g.

image

Alternatively, you can manually update `.changeset/warm-zoos-fry.md`

Comment (mezotv) [reply .changeset/warm-zoos-fry.md]: Yea I edited it should be good now hopefully.

**Review by jrgarciadev — APPROVED** [review]

**Pull request merged**